blocks/math.js

```diff
@@ -82,12 +82,19 @@ Blockly.Blocks['math_positive_number'] = {
    * @this Blockly.Block
    */
   init: function() {
-    this.setHelpUrl(Blockly.Msg.MATH_NUMBER_HELPURL);
-    this.setColour(Blockly.Blocks.math.HUE);
-    this.appendDummyInput()
-        .appendField(new Blockly.FieldNumber('0', 0, Infinity), 'NUM');
-    this.setOutput(true, 'Number');
-    this.setOutputShape(Blockly.OUTPUT_SHAPE_ROUND);
+    this.jsonInit({
+      "message0": "%1",
+      "args0": [
+        {
+          "type": "field_number",
+          "name": "NUM",
+          "min": 0
```

Comment:
Thanks for doing the measurements, @PullJosh! That's helpful to know. Yes - I agree that it's probably not a big deal. However, I think I'm also leaning towards allowing decimals in those blocks. I suspect there is some benefit of disallowing decimals where it doesn't make sense in context (i.e., repeat 1.5 times) to help guide usage, but the connection is not so clear here (where color mappings to numbers are fairly arbitrary anyway).